wallet/src/lib/RapydClient.ts

```diff
@@ -0,0 +1,96 @@
+import axios from "axios";
+import * as Crypto from "crypto-js";
+import moment from "moment";
+import config from "../server/config";
+import {
+  HttpMethods,
+  IRapydClient,
+  IRapydRequestParams,
+  RequestSignature,
+} from "./types";
+
+class RapydClient implements IRapydClient {
```

Comment:
Nice job with this one. 

I think we can keep my PR: https://github.com/aufacicenta/rapydbot/pull/13
And implement `RapydClient` there? 

Since this file is a important part of the `wallet` mcs, it should have Unit tests, but in a separate PR.

Reply:
Sounds good, we can continue with the other one.